lang/rust/Portfile

```diff
@@ -262,164 +262,160 @@ livecheck.regex             refs/tags/(\[\\d\\.\]+).zip
 # cargo from rust-bootstrap is linked against the system's crypto library,
```

Comment:
It appears to me that a few dozen of these dependencies are superfluous, based on these two `Cargo.lock` files.

I recommend adding this comment on how to actually generate the dependencies from the source:

```sh
# use cargo2port:           
# { cargo2port ${worksrcpath}/Cargo.lock ; cargo2port ${worksrcpath}/src/bootstrap/Cargo.lock; } | sed '/^cargo\.crates/d' | sed -E 's/([^\\])$/\1 \\/' | sort -u | sed '1s|^|cargo.crates \\\n|' | sed '$s| \\$||' | pbcopy
```

Here are the diffs that show what I believe are unnecessary dependencies:

<details>
  <summary>diffs</summary>
  
  ### Portfile difference from #22104 using the code above:
  ```diff
--- lang/rust/Portfile	2024-01-14 19:48:06
+++ Portfile		2024-01-14 19:47:38
@@ -105,8 +105,8 @@
 }
 
 patchfiles-append           patch-offline_bootstrap.diff \
-                            patch-emulate_tls.diff
-
+                            patch-emulate_tls.diff \
+                            patch-no_wrong_assembly.diff
 if { ${rust.use_cctools} } {
     patchfiles-append       patch-llvm_libtool.diff
     post-patch {
@@ -258,8 +258,7 @@
 # See: https://github.com/rust-lang/cargo/issues/4813
 # See: https://github.com/mozilla/releases-comm-central/commit/5cd3595203de78a6f47324f53f0cf616f4e383ea
 
-# use cargo2port:
-# { cargo2port ${worksrcpath}/Cargo.lock ; cargo2port ${worksrcpath}/src/bootstrap/Cargo.lock; } | sed '/^cargo\.crates/d' | sed -E 's/([^\\])$/\1 \\/' | sort -u | sed '1s|^|cargo.crates \\\n|' | sed '$s| \\$||' | pbcopy
+# use cargo2port to update it
 # cargo from rust-bootstrap is linked against the system's crypto library,
 # and old systems may not support the required SSL/TLS version to fetch anything from github for example.
 cargo.crates \
@@ -291,11 +290,13 @@
     askama_escape                   0.10.3  619743e34b5ba4e9703bba34deac3427c72507c7159f5fd030aea8cac0cfe341 \
     atty                            0.2.14  d9b39be18770d11421cdb1b9947a45dd3f37e93092cbf377614828a319d5fee8 \
     autocfg                          1.1.0  d468802bab17cbc0cc575e9b053f41e72aa36bfa6b7f55e3529ffa43161b97fa \
+    autocfg                          1.1.0  d468802bab17cbc0cc575e9b053f41e72aa36bfa6b7f55e3529ffa43161b97fa \
     backtrace                       0.3.69  2089b7e3f35b9dd2d0ed921ead4f6d318c27680d4a5bd167b3ee120edb105837 \
     base64                          0.21.2  604178f6c5c21f02dc555784810edfb88d34ac2c73b2eae109655649ee73ce3d \
     basic-toml                       0.1.2  5c0de75129aa8d0cceaf750b89013f0e08804d6ec61416da787b35ad0d7cddf1 \
     bincode                          1.3.3  b1f45e9417d87227c7a56d22e471c6206462cba514c7590c09aff4cf6d1ddcad \
     bitflags                         1.3.2  bef38d45163c2f1dde094a7dfd33ccf595c92905c8f8f4fdc18d06fb1037718a \
+    bitflags                         1.3.2  bef38d45163c2f1dde094a7dfd33ccf595c92905c8f8f4fdc18d06fb1037718a \
     bitflags                         2.4.0  b4682ae6287fcf752ecaabbfcc7b6f9b72aa33933dc23a554d853aea8eea8635 \
     bitflags                         2.4.1  327762f6e5a765692301e5bb513e0d9fef63be86bbc14528052b1cd3e6f03e07 \
     block-buffer                    0.10.2  0bf7fe51849ea569fd452f37822f606a5cabb684dc918707a0193fd4664ff324 \
@@ -313,6 +314,7 @@
     cc                              1.0.73  2fff2a6927b3bb87f9595d67196a70493f627687a71d87a0d692242c33f58c11 \
     cc                              1.0.79  50d30906286121d95be3d479533b458f87493b30a4b5f79a607db8f5d11aa91f \
     cfg-if                           1.0.0  baf1de4339761588bc0619e3cbc0120ee582ebb74b53b4efbf79117bd2da40fd \
+    cfg-if                           1.0.0  baf1de4339761588bc0619e3cbc0120ee582ebb74b53b4efbf79117bd2da40fd \
     chrono                          0.4.26  ec837a71355b28f6556dbd569b37b3f363091c0bd4b2e735674521b4c5fd9bc5 \
     cipher                           0.4.4  773f3b9af64447d2ce9850330c473515014aa235e6a783b02db81ff39e4a3dad \
     clap                             4.4.4  b1d7b8d5ec32af0fadc644bf1fd509a688c2103b185644bb1e29d164e0703136 \
@@ -393,6 +395,7 @@
     env_logger                      0.10.0  85cdab6a89accf66733ad5a1693a4dcced6aeff64602b634530dd73c1f3ee9f0 \
     equivalent                       1.0.0  88bffebc5d80432c9b140ee17875ff173a8ab62faad5b257da912bd2f6c1c0a1 \
     errno                            0.3.5  ac3e13f66a2f95e32a39eaa81f6b95d42878ca0e1db0c7543723dfe12557e860 \
+    errno                            0.3.5  ac3e13f66a2f95e32a39eaa81f6b95d42878ca0e1db0c7543723dfe12557e860 \
     expect-test                      1.4.1  30d9eafeadd538e68fb28016364c9732d78e420b9ff8853fa5e4058861e9f8d3 \
     eyre                             0.6.8  4c2b6b5a29c02cdc822728b7d7b8ae1bab3e3b05d44522770ddd49722eeac7eb \
     fallible-iterator                0.3.0  2acce4a10f12dc2fb14a218589d4f1f62ef011b2d0cc4b3cb1bba8e94da14649 \
@@ -406,6 +409,7 @@
     fluent-langneg                  0.13.0  2c4ad0989667548f06ccd0e306ed56b61bd4d35458d54df5ec7587c0e8ed5e94 \
     fluent-syntax                   0.11.0  c0abed97648395c902868fee9026de96483933faa54ea3b40d652f7dfe61ca78 \
     fnv                              1.0.7  3f9eec918d3f24069decb9af1554cad7c880e2da24a9afd88aca000531ab82c1 \
+    fnv                              1.0.7  3f9eec918d3f24069decb9af1554cad7c880e2da24a9afd88aca000531ab82c1 \
     foreign-types                    0.3.2  f6f339eb8adc052cd2ca78910fda869aefa38d22d5cb648e6485e4d3fc06f3b1 \
     foreign-types-shared             0.1.1  00b0228411908ca8685dba7fc2cdd70ec9990a6e753e89b6ac91a84c40fbaf4b \
     form_urlencoded                  1.2.0  a62bc1cf6f830c2ec14a513a9fb124d0a213a629668a4186f329db21fe045652 \
@@ -436,9 +440,11 @@
     hashbrown                       0.12.3  8a9ee70c43aaf417c914396645a0fa852624801b24ebb7ae78fe8272889ac888 \
     hashbrown                       0.14.2  f93e7192158dbcda357bdec5fb5788eebf8bbac027f3f33e719d29135ae84156 \
     heck                             0.4.1  95505c38b4572b2d910cecb0281560f54b440a19336cbbcb27bf6ce6adc6f5a8 \
+    heck                             0.4.1  95505c38b4572b2d910cecb0281560f54b440a19336cbbcb27bf6ce6adc6f5a8 \
     hermit-abi                       0.3.2  443144c8cdadd93ebf52ddb4056d257f5b52c04d3c804e657d19eb73fc33668b \
     hermit-abi                      0.1.19  62b467343b94ba476dcb2500d242dadbb39557df889310ac77c5d99100aaac33 \
     hex                              0.4.3  7f24254aa9a54b5c858eaee2f5bccdb46aaf0e486a595ed5fd8f86ba55232a70 \
+    hex                              0.4.3  7f24254aa9a54b5c858eaee2f5bccdb46aaf0e486a595ed5fd8f86ba55232a70 \
     home                             0.5.4  747309b4b440c06d57b0b25f2aee03ee9b5e5397d288c60e21fc709bb98a7408 \
     home                             0.5.5  5444c27eef6923071f7ebcc33e3444508466a76f7a2b93da00ed6e19f30c1ddb \
     html5ever                       0.26.0  bea68cab48b8459f17cf1c944c67ddc572d272d9f2b274140f223ecb1da4a3b7 \
@@ -486,6 +492,7 @@
     jsonpath_lib                     0.2.6  61352ec23883402b7d30b3313c16cbabefb8907361c4eb669d990cbb87ceee5a \
     junction                         1.0.0  ca39ef0d69b18e6a2fd14c2f0a1d593200f4a4ed949b240b5917ab51fac754cb \
     lazy_static                      1.4.0  e2abad23fbc42b3700f2f279844dc832adb2b2eb069b2df918f455c4e18cc646 \
+    lazy_static                      1.4.0  e2abad23fbc42b3700f2f279844dc832adb2b2eb069b2df918f455c4e18cc646 \
     lazycell                         1.3.0  830d08ce1d1d941e6b30645f1a0eb5643013d835ce3779a5fc208261dbe10f55 \
     leb128                           0.2.5  884e2677b40cc8c339eaefcb701c32ef1fd2493d71118dc0ca4b6a736c93bd67 \
     levenshtein                      1.0.5  db13adb97ab515a3691f56e4dbab09283d0b86cb45abd991d8634a9d6f501760 \
@@ -499,6 +506,7 @@
     libz-sys                         1.1.9  56ee889ecc9568871456d42f603d6a0ce59ff328d291063a45cbdf0036baf6db \
     linked-hash-map                  0.5.6  0717cef1bc8b636c6e1c1bbdefc09e6322da8a9321966e8928ef80d20f7f770f \
     linux-raw-sys                   0.4.10  da2479e8c062e40bf0066ffa0bc823de0a9368974af99c9f6df941d2c231e03f \
+    linux-raw-sys                   0.4.10  da2479e8c062e40bf0066ffa0bc823de0a9368974af99c9f6df941d2c231e03f \
     litemap                          0.7.1  77a1a2647d5b7134127971a6de0d533c49de2159167e7f259c427195f87168a1 \
     lock_api                        0.4.10  c1cc9717a20b1bb222f333e6a92fd32f7d8a18ddc5a3191a11af45dcbf4dcd16 \
     log                             0.4.17  abb12e687cfb44aa40f41fc3978ef76448f9b6038cad6aef4259d3c095a2382e \
@@ -512,6 +520,7 @@
     md-5                            0.10.5  6365506850d44bff6e2fbcb5176cf63650e48bd45ef2fe2665ae1570e0f4b9ca \
     mdbook                          0.4.31  7b67ee4a744f36e6280792016c17e69921b51df357181d1eb17d620fcc3609f3 \
     measureme                       10.1.1  1930d162935fecd56fc4e0f6729eb3483bac1264542eb4ea31570b86a434b6bc \
+    memchr                           2.5.0  2dffe52ecf27772e601905b7522cb4ef790d2cc203488bbd0e2fe85fcb74566d \
     memchr                           2.5.0  2dffe52ecf27772e601905b7522cb4ef790d2cc203488bbd0e2fe85fcb74566d \
     memmap2                          0.2.3  723e3ebdcdc5c023db1df315364573789f8857c11b631a2fdfad7c00f5c046b4 \
     memoffset                        0.7.1  5de893c32cde5f383baa4c04c5d6dbdd735cfd4a794b0debdb2bb1b421da5ff4 \
@@ -534,6 +543,7 @@
     num_cpus                        1.16.0  4161fcb6d602d4d2081af7c3a45852d875a03dd337a6bfdd6e06407b61342a43 \
     number_prefix                    0.4.0  830b246a0e5f20af87141b25c173cd1b609bd7779a4617d6ec582abaf90870f3 \
     object                          0.32.0  77ac5bbd07aea88c60a577a1ce218075ffd59208b2d7ca97adf9bfc5aeb21ebe \
+    object                          0.32.0  77ac5bbd07aea88c60a577a1ce218075ffd59208b2d7ca97adf9bfc5aeb21ebe \
     odht                             0.3.1  5a518809ac14b25b569624d0268eba1e88498f71615893dca57982bed7621abb \
     once_cell                       1.12.0  7709cef83f0c1f58f666e746a08b21e0085f7440fa6a29cc194d68aac97a4225 \
     once_cell                       1.18.0  dd8b5dd2ae5ed71462c540258bedcb51965123ad7e7ccf4b9a8cafaa4a63576d \
@@ -592,7 +602,9 @@
     rand_xorshift                    0.3.0  d25bf25ec5ae4a3f1b92f929810509a2f53d7dca2f50b794ff57e3face536c8f \
     rand_xoshiro                     0.6.0  6f97cdb2a36ed4183de61b2f824cc45c9f1037f28afe0a322e9fff4c108b5aaa \
     rayon                            1.8.0  9c27db03db7734835b3f53954b534c91069375ce6ccaa2e065441e07d9b6cdb1 \
+    rayon                            1.8.0  9c27db03db7734835b3f53954b534c91069375ce6ccaa2e065441e07d9b6cdb1 \
     rayon-core                      1.12.0  5ce3fb6ad83f861aac485e76e1985cd109d9a3713802152be56c3b1f0e0658ed \
+    rayon-core                      1.12.0  5ce3fb6ad83f861aac485e76e1985cd109d9a3713802152be56c3b1f0e0658ed \
     redox_syscall                    0.3.5  567664f262709473930a4bf9e51bf2ebf3348f2e748ccc50dea20646858f8f29 \
     redox_syscall                   0.2.13  62f25bc4c7e55e0b0b7a1d43fb893f4fa1361d0abe38b9ce4f323c2adfe6ef42 \
     redox_syscall                   0.2.16  fb5a58c1855b4b6819d59012155603f0b22ad30cad752600aadfcb695265519a \
@@ -601,6 +613,7 @@
     regex                            1.8.4  d0ab3ca65655bb1e41f2a8c8cd662eb4fb035e67c3f78da1d61dffe89d07300f \
     regex-automata                   0.2.0  e9368763f5a9b804326f3af749e16f9abf378d227bcdee7634b13d8f17793782 \
     regex-automata                  0.1.10  6c230d73fb8d8c1b9c0b3135c5142a8acee3a0558fb8db5cf1cb65f8d7862132 \
+    regex-automata                  0.1.10  6c230d73fb8d8c1b9c0b3135c5142a8acee3a0558fb8db5cf1cb65f8d7862132 \
     regex-syntax                     0.7.2  436b050e76ed2903236f032a59761c1eb99e1b0aead2c257922771dab1fc8c78 \
     regex-syntax                    0.6.26  49b3de9ec5dc0a3417da371aab17d729997c15010e7fd24ff707773a33bddb64 \
     regex-syntax                    0.6.29  f162c6dd7b008981e4d40210aca20b4bd0f9b60ca9271061b07f78537722f2e1 \
@@ -616,18 +629,22 @@
     rustc_version                    0.4.0  bfa0f585226d2e68097d4f95d113b15b83a82e819ab25717ec0590d9584ef366 \
     rustfix                          0.6.1  ecd2853d9e26988467753bd9912c3a126f642d05d229a4b53f5752ee36c56481 \
     rustix                         0.38.19  745ecfa778e66b2b63c88a61cb36e0eea109e803b0b86bf9879fbc77c70e86ed \
+    rustix                         0.38.19  745ecfa778e66b2b63c88a61cb36e0eea109e803b0b86bf9879fbc77c70e86ed \
     rustversion                     1.0.12  4f3208ce4d8448b3f3e7d168a73f5e0c43a61e32930de3bceeccedb388b6bf06 \
     ruzstd                           0.4.0  ac3ffab8f9715a0d455df4bbb9d21e91135aab3cd3ca187af0cd0c3c3f868fdc \
     ryu                             1.0.10  f3f6f92acf49d1b98f7a81226834412ada05458b7364277387724a237f062695 \
     ryu                             1.0.13  f91339c0467de62360649f8d3e185ca8de4224ff281f66000de5eb2a77a79041 \
     same-file                        1.0.6  93fc1dc3aaa9bfed95e02e6eadabb4baf7e3078b0bd1b4d7b6b0b68378900502 \
+    same-file                        1.0.6  93fc1dc3aaa9bfed95e02e6eadabb4baf7e3078b0bd1b4d7b6b0b68378900502 \
     schannel                        0.1.22  0c3733bf4cf7ea0880754e19cb5a462007c4a8c1914bff372ccc95b464f1df88 \
     scoped-tls                       1.0.1  e1cf6437eb19a8f4a6cc0f7dca544973b0b78843adbfeb3683d1a94a0024a294 \
     scopeguard                       1.1.0  d29ab0c6d3fc0ee92fe66e2d99f700eab17a8d57d1c1d3b748380fb20baa78cd \
+    scopeguard                       1.1.0  d29ab0c6d3fc0ee92fe66e2d99f700eab17a8d57d1c1d3b748380fb20baa78cd \
     security-framework               2.9.1  1fc758eb7bffce5b308734e9b0c1468893cae9ff70ebf13e7090be8dcbcc83a8 \
     security-framework-sys           2.9.0  f51d0c0d83bec45f16480d0ce0058397a69e48fcdc52d1dc8855fb68acbd31a7 \
     self_cell                       0.10.2  1ef965a420fe14fdac7dd018862966a4c14094f900e1650bbc71ddd7d580c8af \
     semver                          1.0.17  bebd363326d05ec3e2f532ab7660680f3b02130d780c299bca73469d521bc0ed \
+    semver                          1.0.17  bebd363326d05ec3e2f532ab7660680f3b02130d780c299bca73469d521bc0ed \
     serde                          1.0.160  bb2f3770c8bce3bcda7e149193a069a0f4365bda1fa5cd88e03bca26afc1216c \
     serde                          1.0.185  be9b6f69f1dfd54c3b568ffa45c310d6973a5e5148fd40cf515acaf38cf5bc31 \
     serde_derive                   1.0.160  291a097c63d8497e00160b166a967a4a79c64f3facdd01cbd7502231688d77df \
@@ -665,10 +682,12 @@
     sysinfo                         0.29.2  9557d0845b86eea8182f7b10dff120214fb6cd9fd937b6f4917714e546a38695 \
     tabled                          0.13.0  4d38d39c754ae037a9bc3ca1580a985db7371cd14f1229172d1db9093feb6739 \
     tar                             0.4.38  4b55807c0344e1e6c04d7c965f5289c39a8d94ae23ed5c0b57aabac549f871c6 \
+    tar                             0.4.38  4b55807c0344e1e6c04d7c965f5289c39a8d94ae23ed5c0b57aabac549f871c6 \
     tempfile                         3.8.0  cb94d2f3cc536af71caac6b6fcebf65860b347e7ce0cc9ebe8f70d3e521054ef \
     tendril                          0.4.3  d24a120c5fc464a3458240ee02c299ebcb9d67b5249c8848b09d639dca8d7bb0 \
     term                             0.7.0  c59df8ac95d96ff9bede18eb7300b0fda5e5d8d90960e76f8e14ae765eedbf1f \
     termcolor                        1.2.0  be55cf8942feac5c765c2c993422806843c9a9a45d4d5c407ad6dd2ea95eb9b6 \
+    termcolor                        1.2.0  be55cf8942feac5c765c2c993422806843c9a9a45d4d5c407ad6dd2ea95eb9b6 \
     terminal_size                    0.3.0  21bebf2b7c9e0a515f6e0f8c51dc0f8e4696391e6f1ff30379559f8365fb0df7 \
     termize                          0.1.1  1706be6b564323ce7092f5f7e6b118a14c8ef7ed0e69c8c5329c914a9f101295 \
     tester                           0.9.1  89e8bf7e0eb2dd7b4228cc1b6821fc5114cd6841ae59f652a85488c016091e5f \
@@ -735,6 +754,7 @@
     valuable                         0.1.0  830b7e5d4d90034032940e4ace0d9a9a057e7a45cd94e6c007832e39edb82f6d \
     vcpkg                           0.2.15  accd4ea62f7bb7a82fe23066fb0957d48ef677f6eeb8215f372f52e48bb32426 \
     version_check                    0.9.4  49874b5167b65d7193b8aba1567f5c7d93d001cafc34600cee003eda787e483f \
+    version_check                    0.9.4  49874b5167b65d7193b8aba1567f5c7d93d001cafc34600cee003eda787e483f \
     walkdir                          2.3.2  808cf2735cd4b6866113f648b791c6adc5714537bc222d9347bb203386ffda56 \
     walkdir                          2.3.3  36df944cda56c7d8d8b7496af378e6b16de9284591917d307c9b4d313c44e698 \
     want                             0.3.1  bfa7760aed19e106de2c7c0b581b509f2f25d3dacaf737cb82ac61bc6d760b0e \
@@ -747,9 +767,13 @@
     wasm-bindgen-shared             0.2.87  ca6ad05a4870b2bf5fe995117d3728437bd27d7cd5f06f13c17443ef369775a1 \
     web-sys                         0.3.61  e33b99f4b23ba3eec1a53ac264e35a755f00e966e0065077d6027c0f575b0b97 \
     winapi                           0.3.9  5c839a674fcd7a98952e593242ea400abe93992746761e38641405d28b00f419 \
+    winapi                           0.3.9  5c839a674fcd7a98952e593242ea400abe93992746761e38641405d28b00f419 \
     winapi-i686-pc-windows-gnu       0.4.0  ac3b87c63620426dd9b991e5ce0329eff545bccbbb34f3be09ff6fb6ab51b7b6 \
+    winapi-i686-pc-windows-gnu       0.4.0  ac3b87c63620426dd9b991e5ce0329eff545bccbbb34f3be09ff6fb6ab51b7b6 \
     winapi-util                      0.1.5  70ec6ce85bb158151cae5e5c87f95a8e97d2c0c4b001223f33a334e3ce5de178 \
+    winapi-util                      0.1.5  70ec6ce85bb158151cae5e5c87f95a8e97d2c0c4b001223f33a334e3ce5de178 \
     winapi-x86_64-pc-windows-gnu     0.4.0  712e227841d057c1ee1cd2fb22fa7e5a5461ae8e48fa2ca79ec42cfc1931183f \
+    winapi-x86_64-pc-windows-gnu     0.4.0  712e227841d057c1ee1cd2fb22fa7e5a5461ae8e48fa2ca79ec42cfc1931183f \
     windows                         0.48.0  e686886bc078bc1b0b600cac0147aadb815089b6e4da64016cbd754b6342700f \
     windows                         0.51.1  ca229916c5ee38c2f2bc1e9d8f04df975b4bd93f9955dc69fabb5d91270045c9 \
     windows-bindgen                 0.51.1  bc1f16b778125675feee0d15d6dd9f6af0e3ac52b3233d63a10aa39230c1cd75 \
@@ -757,6 +781,7 @@
     windows-metadata                0.51.1  753135d996f9da437c0b31dbde3032489a61708361929bcc07d4fba0b161000e \
     windows-sys                     0.45.0  75283be5efb2831d37ea142365f009c02ec203cd29a3ebecbc093d52315b66d0 \
     windows-sys                     0.48.0  677d2418bec65e3338edb076e806bc1ec15693c5d0104683f2efe857f61056a9 \
+    windows-sys                     0.48.0  677d2418bec65e3338edb076e806bc1ec15693c5d0104683f2efe857f61056a9 \
     windows-targets                 0.42.2  8e5180c00cd44c9b1c88adb3693291f1cd93605ded80c250a75d472756b4d071 \
     windows-targets                 0.48.1  05d4b17490f70499f20b9e791dcf6a299785ce8af4d709018206dc5b4953e95f \
     windows-targets                 0.48.5  9a2fa6e2155d7247be68c096456083145c183cbbbc2764150dda45a87197940c \
@@ -785,6 +810,7 @@
     winreg                          0.10.1  80d0f4e272c85def139476380b12f9ac60926689dd2e01d4923222f40580869d \
     writeable                        0.5.3  c0af0c3d13faebf8dda0b5256fa7096a2d5ccb662f7b9f54a40fe201077ab1c2 \
     xattr                            0.2.3  6d1526bbe5aaeb5eb06885f4d987bcdfa5e23187055de9b83fe00156a821fabc \
+    xattr                            0.2.3  6d1526bbe5aaeb5eb06885f4d987bcdfa5e23187055de9b83fe00156a821fabc \
     xz                               0.1.0  3c887690ff2a2e233e8e49633461521f98ec57fbff9d59a884c9a4f04ec1da34 \
     xz2                              0.1.6  c179869f34fc7c01830d3ce7ea2086bc3a07e0d35289b667d0a8bf910258926c \
     xz2                              0.1.7  388c44dc09d76f1536602ead6d325eb532f5c122f17782bd57fb47baeeb767e2 \
  ```
</details>

Reply:
Did you check my comments on the trac ticket?

Reply:
The point is to include a comment in the Portfile with the actual command that generates the dependencies so the next time it’s fast and easy to update the port. No one will ever find it in these comment or trac threads.